artiq/gateware/suservo/iir.py

```diff
@@ -127,7 +127,7 @@ class IIR(Module):
     Each profile stores 8 values, each up to W.coeff bits wide, arranged as:
         [FTW1, B1, POW, CFG, OFFSET, A1, FTW0, B0]
     The lower 8 bits of CFG hold the ADC input channel index SEL.
-    The bits from bit 8 up hold the IIR activation delay DLY.
+    Bits 8:16 hold the IIR activation delay DLY.
```

Comment:
Unclear notation. Is this field 8 bit wide or 9.

Reply:
‘[8:16]’ okay?

Reply:
If you want.
Sure this is nit-picking. But when describing bit ranges in documentation is should be made clear somewhere that python notation and not verilog/vhdl/typical datasheet notation is implied. "The subsequent 8 bits" or "The 8 bits from bit 8 up" is unambiguous.